buildSrc/src/main/resources/META-INF/schemas/guide-metadata.schema.json

```
@@ -0,0 +1,165 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "uri": "https://guides.micronaut.io/schemas",
+  "type": "object",
+  "required": ["title","intro","authors","categories","publicationDate"],
+  "properties": {
+    "minimumJavaVersion": {
+      "type": "integer"
+    },
+    "prerequesites": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "cloud": {
+      "type": "string"
+    },
+    "skipGradleTests": {
+      "type": "boolean"
+    },
+    "intro": {
+      "type": "string"
+    },
+    "skipMavenTests": {
+      "type": "boolean"
+    },
+    "features": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "publicationDate": {
+      "type": "string"
```

```suggestion
      "type": "string"
```

Can we specify in the schema that the publication date is with a format  in YYYY-MM-dd? e.g. 2021-07-13